source/_integrations/kira.markdown

```diff
@@ -129,11 +137,95 @@ repeat:
 
 Some manufacturers (e.g., Samsung) require an IR code to be sent a number of times in a row in rapid succession (usually 3). This doesn't apply to the vast majority of devices, but it can be helpful if needed.
 
+You now need to edit this file to make sure it contains the codes you want the Kira to blast out as a remote or respond to as a sensor. Note that each time you edit and save the kira YAML you need to restart the server for the changes to take effect, reloading the automations alone is not sufficient.
```

Comment:
```suggestion
You now need to edit `kira_codes.yaml` to make sure it contains the codes you want the Kira to blast out as a remote or respond to as a sensor. Note that each time you edit and save the `kira_codes.yaml` file you need to restart home Assistant for the changes to take effect, reloading the automations alone is not sufficient.
```

Reply:
agree with the edit will change to include cap "H" for Home Assistant